ckanext/switzerland/templates/snippets/social.html

```diff
@@ -0,0 +1,6 @@
+{% ckan_extends %}
+{% block social_nav %}
+  <ul class="nav nav-simple">
+    <li class="nav-item"><a href="https://www.linkedin.com/sharing/share-offsite/?url={{ current_url }}" target="_blank"><i class="fa-brands fa-linkedin"></i> LinkedIn</a></li>
```

Comment:
There seems to be an extra space here just in front of the link text:
```suggestion
    <li class="nav-item"><a href="https://www.linkedin.com/sharing/share-offsite/?url={{ current_url }}" target="_blank"><i class="fa-brands fa-linkedin"></i>LinkedIn</a></li>
```

Reply:
thank you for adding it as a suggestion (so useful!) 
I actually took over the blank space from the original template (first commit) and it looks weird (missing spacing between the logo) without it, so I think we should keep it that way :)

Reply:
Thanks for checking! In that case I'm fine with keeping it. :)